src/Routes.tsx

```diff
@@ -57,159 +62,239 @@ export interface ChildProps {
     technicalSchemas: object[];
     economicSchemas: object[];
     hubSchemas: object[];
-  }
+  };
   coaches: any[];
```

Comment:
Would you please create a new interface/types script type for a 'Coach' object for the coaches array? Check the JSON for it in the browsers Network tab/the props object, and create an interface (if it already exists, import it).

Reply:
I THINK I did this, let me know if I did it wrong, Im honestly still pretty sketch on typescript, this is the only place I use it. 😄 I imported Coach and did coaches: Array<Coach>

Reply:
It should look like `coaches: Coach[]`

Reply:
done